src/System Application/App/External File Storage/permissions/FileStorageObjects.PermissionSet.al

```diff
@@ -0,0 +1,13 @@
+namespace System.ExternalFileStorage;
+
+permissionset 9452 "File Storage - Objects"
+{
+    Access = Internal;
+    Assignable = false;
+
+    Permissions =
+        table "File Account" = X,
+        table "File Account Content" = X,
+        table "File Account Scenario" = X,
+        table "File Scenario" = X;
```

Comment:
Add the following codeunits here and remove their inherent entitlement/permissions.

`"External File Storage"`
`"File Account"`
`"File Pagination Data"`
`"File Scenario"`

Reason for this is because they're the facade layers and should still be controlled via permissions.
They also need to be added to the following, otherwise we will get permission issues when we get a SaaS deployment.
The `"File Storage - Read"` should be added to `src/System Application/App/Permissions/SystemApplicationRead.PermissionSet.al`
The `"File Storage - Edit"` should be added to `src/System Application/App/Permissions/SystemApplicationEdit.PermissionSet.al`
The `"File Storage - View"` should be added to `src/System Application/App/Permissions/SystemApplicationView.PermissionSet.al`
The `"File Storage - Objects"` should be added to `src/System Application/App/Permissions/SystemApplicationObjects.PermissionSet.al`

Reply:
@darjoo while you're looking at permissions.
What about entitlements.
a delegated admin has the entitlement for e-mail admin.
Wouldn't it be logical to add the `File Storage Admin` there, too?

https://github.com/microsoft/BCApps/blob/main/src/System%20Application/App/Entitlements/DelegatedAdminagentPartner.Entitlement.al

Reply:
@darjoo I changed the permissions as mentiod, but i have no "File Storage - View". Do wen need one? What is the difference between View and Read? I also added the admin permission to the delegated admin entitelment.

Reply:
You PR is strongly inspired by the Email module, which also doesn't have a view permission set. It's fine not to have it.

If you're curious about the intended structure, look at e.g. https://github.com/microsoft/BCApps/tree/main/src/System%20Application/App/Guided%20Experience/permissions

The idea was to have the following permission set hierarchy:

objects (execute permissions for all the objects in the module)
 |
    read (read permissions to tables in module)
     |
        view (permission needed to view the data of the module, which might include other permissions than read)
         |
            edit (permission needed to edit data of the module)
             |
                admin (permissions needed to setup the module)